### ISSUE TITLE
Not Working For Username and Password

### ISSUE THREAD

**Issue by DADAGAURAV** [byte2mov/gelion-bypass#1]
Not Working For Username and Password

**Comment by DADAGAURAV** [comment]
Valid address found for signature: 48 89 5C 24 20 55 56 57 41 56 41 57 48 8D 6C 24 C9 -> 0x140695820513056
Valid address found for signature: 48 89 5C 24 10 48 89 74 24 18 57 48 81 EC -> 0x140695820487776
Valid address found for signature: 48 89 5C 24 08 48 89 6C 24 10 48 89 74 24 18 57 41 54 41 55 41 56 41 57 48 81 EC 80 02 00 00 -> 0x140695820478640
Keyauth Request Address is valid
Keyauth Error Address is valid
Keyauth Integrity Address is valid
Modify Address is valid
Keyauth Command Error Address is invalid
Keyauth HMAC264 Address is valid
constantTimeStringCompare_address is invalid
Valid address found for signature: 48 89 5C 24 20 55 56 57 41 56 41 57 48 8D 6C 24 C9 -> 0x140695820513056
Valid address found for signature: 48 89 5C 24 10 48 89 74 24 18 57 48 81 EC -> 0x140695820487776
Valid address found for signature: 48 89 5C 24 08 48 89 6C 24 10 48 89 74 24 18 57 41 54 41 55 41 56 41 57 48 81 EC 80 02 00 00 -> 0x140695820478640
Keyauth Request Address is valid
Keyauth Error Address is valid
Keyauth Integrity Address is valid
Modify Address is valid
Keyauth Command Error Address is invalid
Keyauth HMAC264 Address is valid
constantTimeStringCompare_address is invalid

**Comment by byte2mov** [comment]
Yes, to get past username and password you need to modify the json string, in order to do that get a valid keyauth application and log the json and then just input that as your json for the returned json that was hooked.

**Comment by DADAGAURAV** [comment]
can u add me in discord 

username:-deadeyethedev

**Comment by cutypie2351** [comment]
> Yes, to get past username and password you need to modify the json string, in order to do that get a valid keyauth application and log the json and then just input that as your json for the returned json that was hooked.

thats not work its just crashing the app when i do that, can you try that and record if thats work for you?

**Comment by DADAGAURAV** [comment]
> > Yes, to get past username and password you need to modify the json string, in order to do that get a valid keyauth application and log the json and then just input that as your json for the returned json that was hooked.
> 
> thats not work its just crashing the app when i do that, can you try that and record if thats work for you?

i dont even get rn
